README.md

```diff
@@ -189,10 +189,12 @@ _Source:_ [What is Docker](https://www.docker.com/what-docker)
 - [LogJam](https://github.com/gocardless/logjam) - Logjam is a log forwarder designed to listen on a local port, receive log entries over UDP, and forward these messages on to a log collection server (such as logstash) by [@gocardless](https://github.com/gocardless)
 - [Logsene for Docker][spm] Monitoring of Metrics, Events and Logs implemented in Node.js. Integrated [logagent-js](https://github.com/sematext/logagent-js) to detect and parse various log formats. [@sematext][sematext]
 - [Logspout](https://github.com/gliderlabs/logspout) - Log routing for Docker container logs by [@gliderlabs][gliderlabs]
+- [NexClipper](http://www.NexClipper.com) - NexClipper is the container monitoring and performance management solution specialized in Docker, Apache Mesos, Marathon, DC/OS, Mesosphere, Kubernetes by [@Nexcloud](https://github.com/TheNexCloud/NexClipper)
```

Comment:
Change the website to the github repo -> `https://github.com/TheNexCloud/NexClipper`

Reply:
I changed url to GitHub repo